frontend/src/pages/accounts/settings.module.scss

```diff
@@ -31,13 +65,23 @@
   color: $color-dark-gray-50;
   font-weight: 400;
   padding-bottom: $spacing-md;
+  flex-basis: 0;
```

Comment:
This fixes the tablet view where there's a forced gap between the label and the checkbox: 

<img width="834" alt="image" src="https://user-images.githubusercontent.com/2692333/167020515-0afcb44b-c8f6-46d6-a153-cf412c63da60.png">

On desktop, the gap returns: 
<img width="1220" alt="image" src="https://user-images.githubusercontent.com/2692333/167020561-23e21841-3a6c-4f35-8f00-2cf1f01dc27d.png">